pyproject.toml

```diff
@@ -7,11 +7,10 @@ license = "GPL-3.0-only"
 readme = "README.md"
 packages = [
     {include = "runzi"},
-    {include = "scripts"}
 ]
 
 [tool.poetry.scripts]
-runzi = 'scripts.cli:rnz'
+runzi-hazard = 'runzi.automation.cli:rnz_hazard'
```

Comment:
@chrisdicaprio did you name it rnz-hazard becuase that's all it will ever support, and other stuff gets it's own cli? Ifg so maybe this `cli.py` should be `hazazd_cli.py`?